checker/manual/external-tools.tex

```diff
@@ -797,15 +797,26 @@ \section{tIDE\label{tide}}
 
 \section{NetBeans\label{netbeans}}
 
-To run a checker within NetBeans:
+
+There are two approaches to running a checker in Netbeans, via modifying the project properties, or via a custom ant target. 
+
+
+\subsection {Adding a checker via the project properties\label{netbeans-project-properties}}
```

Comment:
For consistency with the rest of the document, don't insert an extra space after "subsection".